cmd/debos/debos.go

```diff
@@ -18,9 +18,17 @@ func bailOnError(err error, a debos.Action, stage string) {
 		return
 	}
 
-	log.Fatalf("Action `%s` failed at stage %s, error: %s", a, stage, err)
+	log.Printf("Action `%s` failed at stage %s, error: %s", a, stage, err)
+	panic(code1)
 }
 
+type exitcode uint
+
+const (
+	code0 exitcode = 0
+	code1          = 1
```

Comment:
I'm not sure we need these, but I'd at least try to find more meaningful names here.

Reply:
Useful for panic( but do not need it anymore